### ISSUE TITLE
Accessibility - sufficient color contrast against the background

### ISSUE THREAD

**Issue by benamar19-b** [DependencyTrack/frontend#1624]
### Current Behavior

Text (Date in the graphs, Graph subheadings, Titles of the portfolio statistics) in the dashboard page has poor colour contrast. 
WCAG requires a minimum ratio of 4.5:1 for small text.

<img width="409" height="166" alt="Image" src="https://github.com/user-attachments/assets/d40ad965-1517-4c7d-a5eb-0415c4a30fc4" />

### Proposed Behavior

Contrast  text in Text (Date in the graphs, Graph subheadings, Titles of the portfolio statistics)   with the background should be 4.5.1 minimum

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/main/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/frontend/issues) for whether this enhancement was already requested
